protected/config/core/common.php

```diff
@@ -321,7 +321,7 @@
                 'uri' => 'dashboard/cataract',
                 'position' => 4,
                 'userrule' => 'isSurgeon',
-                'restricted' => array('admin'),
+                'restricted' => array('admin,report'),
                 'options' => array('target' => '_blank'),),
```

Comment:
This change doesn't seem related to the fix. Can you explain its purpose?

Reply:
Oh, this related to the admin/cataract audit page, which seems request report role to show the page.